django/db/models/query.py

```diff
@@ -971,9 +971,9 @@ def update_or_create(self, defaults=None, create_defaults=None, **kwargs):
         specifying whether an object was created.
         """
         if create_defaults is None:
-            update_defaults = create_defaults = defaults or {}
-        else:
-            update_defaults = defaults or {}
+            create_defaults = defaults or {}
+
+        update_defaults = defaults or {}
```

Comment:
It would seem to read better as -- _if `create_defaults` is not specified, use the same value as `update_defaults`_:

```python
        update_defaults = defaults or {}
        if create_defaults is None:
            create_defaults = update_defaults
```

Reply:
Done. Thank for review.